### PR TITLE
Applying concepts in a clone of a New York Times article page.

### PR DESCRIPTION
**Pull request by kenderb**
The page contains the next mains tags:

- A nav tag for the navbar.
- Main tag and inside an article tag.
- We create aside tag for the editor picks.
- And a foot tag that can be displayed on the hole page.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,716 @@
...
+        <li><button class="menu-btn" type="button">
+            <ion-icon name="menu-outline" class="ion-icon"></ion-icon>
+          </button></li>
+        <li><button type="button" class="magnifying-glass-btn">
+            <ion-icon name="search-outline" class="ion-icon"></ion-icon>
+          </button></li>
+        <li><a href="#" class="space-cosmos-link">SPACE & COSMOS
+          </a></li>
+        <li>
+          <img src="https://upload.wikimedia.org/wikipedia/commons/5/58/NewYorkTimes.svg" alt="TNYT-icon" />
```
Comment (krys2fa) [inline index.html]: - [x] [REQUIRED] The NY Times logo should be a link ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/positioning_and_floating_requirements.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538403/))

File: index.html [hunk 2]
```diff
@@ -0,0 +1,716 @@
...
+              <p>A New Ship’s Mission: Let the Deep Sea Be Seen</p>
+              <p>Sept. 21</p>
+            </div>
+            <div class="pick-info">
+              <img src="assets/images/more-articule-images/editors3.jpg" alt="editors3" />
+              <p>Tyler Schiffman</p>
+              <p>Bringing the Ocean’s Midnight Zone Into the Light</p>
+              <p>Sept. 22</p>
+            </div>
+          </div>
```
Comment (krys2fa) [inline index.html]: - [x] [REQUIRED] These articles with image previews should be links ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/positioning_and_floating_requirements.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538403/))

**Review by krys2fa — CHANGES_REQUESTED** [review]
## Awesome work @carloso0114 and @kenderb 👏 
Your project displays good use of semantic HTML as well as CSS classes. 💪 
It also has a professional README and a good title and description in the Pull Request. 👍 

Just a few more adjustments to complete this project. They can be seen below and as well as in the comments.

NB: This review was done at a screen size of 1440px.
### Status: Changes Required ♻️ 

- [REQUIRED ] Your git commit messages can be improved by writing them in the imperative tense. Find out more [here](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/).

- [REQUIRED] Some of the images in the "Theory of Inflation" section are not showing. Please fix this.
Hint: Check the extensions of the images with those you have in your HTML.
Screen Shot 2020-09-29 at 1 38 22 AM

- [x] [REQUIRED] The "Editor's Picks" section needs some margin to it's left. Please fix this.
Screen Shot 2020-09-29 at 1 38 05 AM



That's all. Please request for another review after implementing the changes.

Happy Coding 🖥️  😃 

###### You can contact me on slack (Chris Amanor) for any clarification.

**Review by javitocor — CHANGES_REQUESTED** [review]
#STATUS CHANGES REQUIRED ♻️ 

Good job team with the implemented changes! 👍 
You have done a great job and the website looks amazing! 💯 
Linters are green 🟢  and your code is clean and clear 👏 

However, still a few changes to go! 
Please, check my comments below 👇  and once the changes are done request a new code review from your dashboard ⏩ 

Good luck, team 😃 

[REQUIRED]
- The editor's picks still need a bit more of margin, in big screens looks good enough, but under 1200px still needs some fixing, please fix it
Capture

[REQUIRED]
- in the More in Space and Astronomy section you made a good job adding images as a tag, kindly do the same with the text under the pics.

[OPTONAL]
- You can use `section` tag inside your `main`, it will make your code more readable

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle:
## :wave: :wave: Hello Team!! Beautiful work!! :clap: :clap:

I liked the similarities with the original website.

## Required Changes.
### [Mandatory] :red_circle: 

- [x] MIT Licence link is broken in README.

- [x] Alternative text attribute should descriptive. Example
**Actual**: ```alt="Aside1"```
**Improved** :+1: : ```alt="Arthur S. Comey"```

- [x] Footer NYT logo should be a link
image

### [OPTIONAL]  :large_blue_circle: 

- [ ] Prefer CSS custom selector over general CSS selectors. Custo selectors are more specific and easier to identify. This also helps with readability. Example:
**Actual**: 
```
nav li:nth-of-type(5) {
  margin-left: auto;
}
```
**Improved**
```
.nav__item {
  margin-left: auto;
}
```

- [ ] Keep the code DRY. Example:
**Actual**:
```
nav li:nth-of-type(5) {
  margin-left: auto;
}

nav li:nth-of-type(4) {
  margin-left: auto;
}
```
**Improved** v1 :+1: :
```
nav li:nth-of-type(5), 
nav li:nth-of-type(4) {
  margin-left: auto;
}
```
**Improved** v2 :+1: :
```
.nav__item {
  margin-left: auto;
}
```

- [ ] Follow this example when embedding an Image.
```
<figure>
  <img src="pic_trulli.jpg" alt="Trulli" style="width:100%">
  <figcaption>Fig.1 - Trulli, Puglia, Italy.</figcaption>
</figure>
```

### [Comments] :green_circle: 

## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

**Review by jubaan — APPROVED** [review]
# Status: Approved :tada: :tada: :tada:

## Congrats team!! You may now merge to master! 

## See you at the top :rocket: !!!!!

**Pull request merged**